### PR TITLE
[Feature][AES-1933][1822] Fix image display issue in Carousel and logic for Zaph font type and Korean regions

### PR DESCRIPTION
**Pull request by beau-charman**
This PR addresses the Image display issue in the Carousel, raised in https://aesoponline.atlassian.net/browse/AES-1933

it also adds some conditional logic around the Zaph font type and Korean regions: https://aesoponline.atlassian.net/browse/AES-1822

### CODE REVIEW

File: src/styles/mixins/typography.css [hunk 1]
```diff
@@ -8,3 +8,12 @@
   line-height: 1.2;
   text-size-adjust: 100%;
 }
+
+%zaph-serif-typography {
+  font-family: var(--font-family-zapf-serif);
+
+  html[lang='en'] & {
```
Comment (zack76) [inline src/styles/mixins/typography.css:15]: should **_Suisse_** be displayed in **KR** site?

Comment (hassanraza0054) [reply src/styles/mixins/typography.css:15]: Yup same question. To build on what Zack is asking, should the following be used instead?

image

Comment (beau-charman) [reply src/styles/mixins/typography.css:15]: still working on this one ya'll

Comment (beau-charman) [reply src/styles/mixins/typography.css:15]: I will add reviewers once it is complete

Comment (beau-charman) [reply src/styles/mixins/typography.css:15]: > Yup same question. To build on what Zack is asking, should the following be used instead?

Sorry, I don't understand the question

**Comment by beau-charman** [comment]
Okay ready for review

**Review by zack76 — APPROVED** [review]

**Pull request merged**